### PR TITLE
added a new task list item

### PR DESCRIPTION
**Pull request by samMcCarthyBJSS**
Story [AB#5436](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/5436)
Task [AB#8593](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/8593)

image

### CODE REVIEW

File: app/pages/task-list/existingorder/ui-tests/ui.test.js [hunk 1]
```diff
@@ -755,6 +755,160 @@ test('should only render the complete tag for task 7 item 1 when returned as com
...
+  await pageSetup(t, true, mockOrderSummary);
+  await t.navigateTo(pageUrl);
+
+  const task8Item1 = Selector('li[data-test-id="task-7-item-0"]');
+
+  await t
+    .expect(task8Item1.find('a').exists).ok()
+    .click(task8Item1.find('a'))
+    .expect(getLocation()).eql(`http://localhost:1234${baseUrl}/organisation/order-id/funding-sources`);
+});
```
Comment (alibombaye) [inline app/pages/task-list/existingorder/ui-tests/ui.test.js:888]: Nice.. Awesome test descriptions 👍

File: app/pages/task-list/existingorder/ui-tests/ui.test.js [hunk 2]
```diff
@@ -755,6 +755,160 @@ test('should only render the complete tag for task 7 item 1 when returned as com
...
+test('should only render the complete tag for task 7 item 1 when returned as complete from the API', async (t) => {
+  const mockOrderSummary = generateMockOrderSummary([{ id: 'associated-services', status: 'complete' }]);
+  await pageSetup(t, true, mockOrderSummary);
+  await t.navigateTo(pageUrl);
+
+  const task7Item1CompleteTag = Selector('[data-test-id="task-6-item-0-complete-tag"]');
+
+  await t
+    .expect(task7Item1CompleteTag.exists).ok();
+});
```
Comment (alibombaye) [inline app/pages/task-list/existingorder/ui-tests/ui.test.js:910]: bit confused by the above 2?

Comment (samMcCarthyBJSS) [reply app/pages/task-list/existingorder/ui-tests/ui.test.js:910]: ah forgot to update these to task 8

File: app/pages/task-list/taskListManifest.json [hunk 3]
```diff
@@ -114,6 +114,34 @@
...
+            },
+            {
+              "statusDependencies": ["description", "ordering-party", "supplier", "commencement-date", "service-recipients", "catalogue-solutions", "associated-services"],
+              "countDependencies": ["service-recipients", "associated-services"],
+              "zeroCountDependencies": ["catalogue-solutions"]
+            },
+            {
+              "statusDependencies": ["description", "ordering-party", "supplier", "commencement-date", "service-recipients", "catalogue-solutions", "associated-services"],
+              "zeroCountDependencies": ["catalogue-solutions", "associated-services"]
+            }
```
Comment (alibombaye) [inline app/pages/task-list/taskListManifest.json:141]: So glad we engineered this in. Imagine code if we did it with just if,elses in the code base

Comment (samMcCarthyBJSS) [reply app/pages/task-list/taskListManifest.json:141]: yeah defintiely was an easy addition too!

**Pull request merged**